lib/morse_translator.rb

```diff
@@ -1,6 +1,9 @@
 class MorseTranslator
 
-  def initialize
+  attr_reader :diccionary,
```

Comment:
You don't need an attr_reader since you are accessing the instance variable from within the class

Reply:
The comma on this line is causing an error